Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -1589,8 +1586,7 @@ static void inverse_dct(JPEGLoadingContext const& context, Vector<Macroblock>& m
 static void undo_subsampling(JPEGLoadingContext const& context, Vector<Macroblock>& macroblocks)
 {
     // The first component has sampling factors of context.sampling_factors, while the others
-    // are either 1x1 or for the 4th component also context.sampling_factors. See
-    // read_start_of_frame() which currently enforces these restrictions.
+    // are divide the first component's sampling factors. This is enforced by read_start_of_frame().
```

Comment:
Is it me not understanding English? "are divide" doesn't seem correct.

Reply:
That was just wrong, thanks :)